client/components/registration/subscriptions.test.js

```diff
@@ -109,25 +122,57 @@ describe("test subscriptions", () => {
     );
     wrapper = initShallow(props);
     wrapper.instance().setState({plans, plansFetched: true});
-    expect(wrapper.find("input[name='plan_selection']").length).toBe(2);
+    expect(wrapper.find("input[name='plan_selection']").length).toBe(3);
     expect(lastConsoleOutuput).toBe(null);
   });
 
-  it("show billing info only when verifies_identity is true", () => {
+  it("should not show billing info when requires_payment is false", () => {
     axios.mockImplementationOnce(() =>
       Promise.resolve({
         status: 201,
         statusText: "ok",
         data: plans,
       }),
     );
+    props.settings.mobile_phone_verification = true;
     wrapper = initShallow(props);
     wrapper.instance().setState({plans, selected_plan: 0, plansFetched: true});
     expect(wrapper.find(".billing-info").length).toBe(0);
     expect(wrapper.find("input[name='username']").length).toBe(0);
-    wrapper.instance().setState({selected_plan: 1});
+    // phone_number field should be rendered since this plan does not verifies identity
+    expect(wrapper.find("input[name='phone_number']").length).toBe(1);
```

Comment:
```suggestion
    expect(wrapper.find("t[name='phone_number']").length).toBe(1);
```
When I debugged it the wrapper is showing a custom tag (`t`) for the phone input field @pandafy, So replacing it with that passes the test.

Reply:
Thank you Sankalp! After making this change, the test suite passes. 😄